_posts/2021-10-04-insertion-sort.md

```diff
@@ -0,0 +1,123 @@
+---- 
+  title:  "Insertion Sort"
+  description:  "Insertion sort is a simple sorting technique. Here in this post, first we will see the algorithm, then we will see an example of its working, we will also have a look on its implementation in C++ language, then in the last we will discuss its other characteristics."
+  category: dsa
+  image: /assets/images/categories/algorithm.jpg
+  keywords: # Keywords your want to ad for SEO

```

Comment:
'Keywords your want to an ad for SEO' (remove this line and directly start with your keywords)